sentry-kotlin-extensions/src/test/java/io/sentry/kotlin/SentryContextTest.kt

```diff
@@ -16,6 +16,10 @@ import kotlin.test.assertNull
 
 class SentryContextTest {
 
+    // TODO [HSM] In global hub mode SentryContext behaves differently
```

Comment:
We should specify how Sentry coroutines support is supposed to work for Android (globalHubMode) and then check if/what needs to be changes.